packages/angular-meteor-data/modules/view-model.js

```diff
@@ -0,0 +1,90 @@
+angular.module('angular-meteor.view-model', [
+  'angular-meteor.utils',
```

Comment:
I'm going through angular-meteor line by line so it came up. Good job @ozsay with 1.3.6! :)